### PR TITLE
Converted Auth samples to use auth grant flow instead of implicit grant flow

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Comment by ChetanSharma-msft** [comment]
Execute Build Pipeline:

**Comment by Pawan-MSFT** [comment]
Executed Build Pipeline:
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3821665809

Result:
image

**Comment by ChetanSharma-msft** [comment]
@Prithvi-MSFT - Please execute build pipeline once comments fixes are done.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments on your PR.

**Comment by ChetanSharma-msft** [comment]
Execute build pipeline:

**Comment by Prithvi-MSFT** [comment]
Build Pipeline status  - All builds passed.
image

File: samples/app-complete-auth/csharp/AppCompleteAuth/Pages/auth-end.cshtml.cs [hunk 1]
```diff
@@ -1,11 +1,34 @@
...
+
+		/// <summary>
+		/// Model entity for storing clientId of the app.
+		/// </summary>
+        public string ClientId { get; set; }
+
+		/// <summary>
+		/// Initialize the AuthEnd Class.
+		/// </summary>
+        public authEnd(IConfiguration configuration)
```
Comment (ChetanSharma-msft) [inline samples/app-complete-auth/csharp/AppCompleteAuth/Pages/auth-end.cshtml.cs:21]: Summary alignment is not looks correct.

File: samples/app-complete-auth/csharp/AppCompleteAuth/Pages/auth-start.cshtml.cs [hunk 2]
```diff
@@ -5,18 +5,30 @@ namespace AppCompleteAuth.Pages
...
         private readonly IConfiguration _configuration;
 
+		/// <summary>
+		/// Model entity for storing clientId of the app.
+		/// </summary>
         public string ClientId { get; set; }
 
+		/// <summary>
+		/// Initialize the AuthStart Class.
+		/// </summary>
```
Comment (ChetanSharma-msft) [inline samples/app-complete-auth/csharp/AppCompleteAuth/Pages/auth-start.cshtml.cs:20]: Summary alignment doesn't looks correct.

File: samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-end.cshtml.cs [hunk 3]
```diff
@@ -1,11 +1,34 @@
...
+		/// </summary>
+        public string ClientId { get; set; }
+
+		/// <summary>
+		/// Initialize the AuthEnd Class.
+		/// </summary>
+        public authEnd(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
```
Comment (ChetanSharma-msft) [inline samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-end.cshtml.cs:24]: Summary alignment doesn't looks correct.

File: samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-start.cshtml.cs [hunk 4]
```diff
@@ -5,18 +5,29 @@ namespace StaggeredPermission.Pages
...
+		/// </summary>
         public authStart(IConfiguration configuration)
         {
             _configuration = configuration;
         }
 
+		/// <summary>
+		/// Handler method called when get request is made to auth-start page.
+		/// </summary>
         public void OnGet()
```
Comment (ChetanSharma-msft) [inline samples/tab-staggered-permission/csharp/StaggeredPermission/Pages/auth-start.cshtml.cs:29]: Summary alignment doesn't looks correct.

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comments.

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

**Review by Pawan-MSFT — APPROVED** [review]
looks fine, approving!

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Prithvi-MSFT** [comment]
Build pipeline status - All builds passed
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/3828140072

image

**Pull request merged**